### PR TITLE
Open-source hyphens fix

### PR DESCRIPTION
**Pull request by ticklemyIP**
Changes proposed in this PR:

- There were a few instances of "open source" with no hyphens along the codebase, I fixed those. I have NOT added hypens to "Android Open Source Project" strings.

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

**Comment by dngray** [comment]
@jonaharagon curious to know what your thoughts are on this if, it should have a hyphen or not.

**Comment by ticklemyIP** [comment]
> @jonaharagon curious to know what your thoughts are on this if, it should have a hyphen or not.

All of the other `open-source` stuff way hyphenated that's why I think it should be at least uniform.

**Comment by jonaharagon** [comment]
What a fascinating question, we really need a style guide lol 🙂 

My thought is that open source is hyphenated as a compound adjective when it *precedes* a noun (i.e. "this is open-source software), and not hyphenated otherwise (i.e. "Privacy Guides is open source"). I pushed another commit here which standardizes this across the website to see if we like it.

**Review by dngray — COMMENTED** [review]
I'm not sure that open source has a hyphen in most places on the internet these days.

File: docs/about/criteria.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ We do not make money from recommending certain products, we do not use affiliate
 We apply these priorities when considering new recommendations:
 
 - **Secure**: Tools should follow security best-practices wherever applicable.
-- **Source Availability**: Open source projects are generally preferred over equivalent proprietary alternatives.
+- **Source Availability**: Open-source projects are generally preferred over equivalent proprietary alternatives.
```
[the remark this answers is not in the document]
Comment (jonaharagon) [reply docs/about/criteria.md]: Wouldn't that be correct given the style we established? "Open-source" is an adjective modifying "projects" in this case.

**Review by jonaharagon — APPROVED** [review]

**Comment by jonaharagon** [comment]
@ph00lt0 read the standard again lol:

> **Hyphenate** open-source as an adjective preceding a noun, as in *open-source software*.

> **Don't hyphenate** in other instances, such as *Open source is a development model in which ...*

**Comment by paulverbeke** [comment]
> https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

too bad microsoft didn't make this page for french style ahah now I have to decide again by myself ^^'

**Pull request merged**